Formula/civl.rb

```diff
@@ -0,0 +1,22 @@
+
+class Civl < Formula
+  desc "The Concurrency Intermediate Verification Language"
+  homepage "http://vsl.cis.udel.edu/civl/"
+  url "http://vsl.cis.udel.edu/lib/sw/civl/1.7/r3157/release/CIVL-1.7_3157.tgz"
+  sha256 "33fa5b4877fbf494b0a4b426c15d6a165fa3db48a9f97cbf391818d965d2dfb9"
+  depends_on :java => "1.8+"
+  depends_on "z3"
+  depends_on "cvc4"
+
+  def install
+    libexec.install "lib/civl-1.7_3157.jar"
+    bin.write_jar_script libexec/"civl-1.7_3157.jar", "civl"
+    share.install "doc"
+    share.install "examples"
+    share.install "licenses"
+    share.install "emacs"
+  end
+  test do
+    system "#{bin}/civl"
```

Comment:
Yep! In fact, definitely leave a comment when you think this is ready for another review. We use the "needs response" label to indicate that we're waiting for a response or update from the submitter. When you leave a comment on the PR, that label will go away and we'll notice that this PR needs attention and get back to you.

Thanks for your contribution and for working with us on this!